manifests/systemd.pp

```diff
@@ -0,0 +1,20 @@
+class profiles::systemd (
+  String $system_max_use   = '500M'
+) inherits ::profiles {
+  include systemd
+
+  systemd::journald { 'SystemMaxUse':
```

Comment:
Ik denk dat die systemd::journald class private is, dus niet rechtstreeks kan opgeroepen worden.
Volgens mij moet je gewoon de main class gebruiken: `class { systemd: journald_settings => { SystemMaxUse => $system_max_use } }`